### ISSUE TITLE
A11y_CDE - Mission Control and APIs_ Driver Shiprom Search_Screen Reader: Narrator is not announcing about the selected item in the 'Field' combo box  

### ISSUE THREAD

**Issue by Bhaskar301101** [facebook/react#22227]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier

Environment Details:
URL:  https://mission-control-ci.azurewebsites.net/
OS: Windows 10
Version: 21H1 (OS Build 19043.1052)
Browser: Version 93.0.916.1 (Edge Dev)

Note: Same issue is observed in "Released  Driver Search"

Repro Steps:​​
1) Turn on screen reader
2)  Hit the application URL : https://mission-control-ci.azurewebsites.net/
3) Tab till the windows menu button in left navigation and press enter
4) Tab till "Driver Operations" link under "Drivers and Apps" and press enter
5) Tab till "Driver Shiproom" tab and press enter
6) Tab till 'Field' Combo box
7) Verify whether the narrator is announcing about the selected item form the combo box or not

Actual Result:​​
Narrator is not announcing the pre-selected item in the 'Field' combo box instead reading as group when the focus lands on the combo box

Expected Result:​​
 Narrator ​should read the pre-selected option “Driver Id” when the focus lands on the “Field” combo box

User Impact:​
Screen reader users will not know which option is pre-selected item if the screen reader doesn’t read it out to the user.

MAS Reference:​​
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
<img width="590" alt="Bug3" src="https://user-images.githubusercontent.com/89897257/131653885-7244ca7d-1465-4d76-8b36-eacbf2fb8e8a.png">

**Comment by eps1lon** [comment]
This repository is not affiliated with https://mission-control-ci.azurewebsites.net/%E2%80%8B
